ApplyToBecomeInternal/ApplyToBecomeInternal/Pages/TaskList/LegalRequirements/LegalConsultation.cshtml.cs

```diff
@@ -31,9 +31,15 @@ public async Task<IActionResult> OnPostAsync(int id)
 				nameof(ThreeOptions.NotApplicable) => ThreeOptions.NotApplicable,
 				_ => LegalRequirements.ConsultationDone
 			};
-
 			await LegalRequirementsRepository.UpdateByProjectId(id, LegalRequirements);
 
+			var (returnPage, fragment, back) = GetReturnPageAndFragment();
```

Comment:
Could move this into the LegalModelBase.cs

Reply:
True!